resources/js/Pages/Galleries/hooks/useWalletDraftGalleries.test.ts

```diff
@@ -0,0 +1,83 @@
+import { expect, type SpyInstance } from "vitest";
+import { useWalletDraftGalleries } from "./useWalletDraftGalleries";
+import * as AuthContextMock from "@/Contexts/AuthContext";
+import { renderHook, waitFor } from "@/Tests/testing-library";
+
+let useAuthSpy: SpyInstance;
+
+vi.mock("@/Contexts/AuthContext", () => ({
+    useAuth: () => ({ wallet: { address: "mockedWalletAddress" } }),
+}));
+
+const defaultGalleryDraft = {
+    id: 1,
+    walletAddress: "mockedAddress",
+    nfts: [],
+    title: "",
+    cover: null,
+    coverTye: null,
+    updatedAt: 169901639000,
+};
+
+const indexedDBMocks = {
+    add: vi.fn(),
+    getAll: vi.fn().mockResolvedValue([]),
+    update: vi.fn(),
+    deleteRecord: vi.fn(),
+    openCursor: vi.fn(),
+    getByIndex: vi.fn(),
+    clear: vi.fn(),
+    getByID: vi.fn().mockResolvedValue(defaultGalleryDraft),
+};
+
+const mocks = vi.hoisted(() => ({
+    useIndexedDB: () => indexedDBMocks,
+}));
+
+vi.mock("react-indexed-db-hook", () => ({
+    useIndexedDB: mocks.useIndexedDB,
+}));
+
+describe("useWalletDraftGalleries", () => {
+    beforeAll(() => {
+        useAuthSpy = vi.spyOn(AuthContextMock, "useAuth").mockReturnValue({
```

Comment:
I don't think you need this spy instance as you are already passing `address` as argument.